ldi-orchestrator/ldio-connectors/ldio-http-in/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/LdioHttpIn.java

```diff
@@ -21,11 +23,13 @@ public LdioHttpIn(ComponentExecutor executor, LdiAdapter adapter, String endpoin
 	public RouterFunction<ServerResponse> mapping() {
 		return route(POST("/%s".formatted(endpoint)),
 				req -> {
-					String contentType = req.headers().contentType().orElseThrow().toString();
+					String contentType = req.headers().contentType()
+							.orElseThrow(() -> new NoSuchElementException("No Content-Type header found"))
+							.toString();
 					return req.bodyToMono(String.class)
 							.doOnNext(content -> getAdapter().apply(Content.of(content, contentType))
 									.forEach(getExecutor()::transformLinkedData))
-							.flatMap(body -> ServerResponse.noContent().build());
+							.flatMap((body) -> ServerResponse.ok().build());
```

Comment:
I wonder if http 202 (accepted) wouldn't be the best fit here? Since we cannot give any feedback on the processing of the entire pipeline.

Reply:
202 Accepted
    The request has been accepted for processing, but the processing has not been completed. The request might or might not be eventually acted upon, and may be disallowed when processing occurs.

Reply:
This feedback came from @WLefever-Cegeka .. 
Technically, we do give feedback that the adapter succesfully interpreted the message and sent it to the transformer pipeline.. so ok might not be such a bad move?

Reply:
Yes but async processing will happen later and the request may still be disallowed by a transformer. For example, posting a valid json-ld with invalid geojson-ld in it would be allowed by the http-in but the geojson-to-wkt transformer will deny the invalid geojson. I feel like 202 better reflects this. But since you went for 204, Wouter for 200 and I 202. Any of those is probably ok.

Reply:
We could always go for the Teapot status code, but it's in the wrong range :p https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/418